docs/New-Maintainer-Checklist.md

```diff
@@ -83,17 +92,17 @@ If a maintainer is elected to the Homebrew's [Technical Steering Committee](http
 - Invite them to the [**@Homebrew/tsc** team](https://github.com/orgs/Homebrew/teams/tsc/members)
 - Make them [billing managers](https://github.com/organizations/Homebrew/settings/billing) and [moderators](https://github.com/organizations/Homebrew/settings/moderators) on the Homebrew GitHub organisation
 
-When they step down as a TSC member, revoke or downgrade their access to all of the above.
+When they cease to be a TSC member, revoke or downgrade their access to all of the above.
 
-## Owners
+## System Owners
```

Comment:
Dunno how much we've linked to this, probably not much. If potentially we have, then it'd be in bounds to shove this in there:

```suggestion
## System Owners
<a name="owners"></a>
```

Reply:
I'd be tempted to just leave the naming as-is unless you have some reasoning to change?

Reply:
It's to disambiguate between "owners of Homebrew" which a naive reader might imply versus "owners of systems that Homebrew uses" as a necessary operations detail.

Reply:
Agreed with @p-linnane.

Reply:
```suggestion
## Owners
```